lua/nvim-treesitter/parsers.lua

```diff
@@ -11,6 +11,8 @@ for ft, lang in pairs {
   ["typescript.tsx"] = "tsx",
   ["terraform-vars"] = "terraform",
   ["html.handlebars"] = "glimmer",
+  ["javascript.glimmer"] = "glimmer_javascript",
```

Comment:
You are duplicating the information in the parsers table. This is only needed for _additional_ filetypes.